docs/vocabulary.md

```diff
@@ -51,6 +52,29 @@ humans to communicate and collaborate better.
 This section contains list of some key terms used by various CI/CD tools and
 technologies.
 
+### ArgoCD
+
+[ArgoCD](https://argoproj.github.io/argo-cd/) is a declarative, GitOps continuous delivery tool for Kubernetes.
+
+Some of the core ArgoCD concept are listed below: [[16]]
+
+- **Application**: A group of Kubernetes resources as defined by a manifest. This is a Custom Resource Definition (CRD).
+- **Application controller** [[19]]: Kubernetes controller which continuously monitors running applications and compares live vs. desired state.
+- **Application source type**: Which Tool is used to build the application.
```

Comment:
Tool should be lowercase t.

Reply:
Switched to lower case, I believe the intent of the uppercase (as used in the original reference) is to denote that the term is also one defined in the list, I don't think lower case makes its any less comprehensible though, and it does _look_ better.